instrumentation/java-concurrent/akka-2.5-testing/akka-2.5-testing.gradle

```diff
@@ -4,12 +4,8 @@ ext {
 }
 
 apply from: "$rootDir/gradle/instrumentation.gradle"
-apply from: "$rootDir/gradle/test-with-scala.gradle"
```

Comment:
can u hold off on this scala-related cleanup? there's big changes coming shortly in https://github.com/DataDog/dd-trace-java/pull/1625

Reply:
Yeah, that crept here semi-accidentally, because my builds failed for some mysterious reasons :( Will try to revert